pkg/lvgl/contrib/lvgl.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2019 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_lvgl
+ * @{
+ *
+ * @file
+ * @brief       LittlevGL glue code
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @}
+ */
+
+#include "thread.h"
+
+#include "xtimer.h"
+#include "log.h"
+
+#include "lvgl/lvgl.h"
+#include "lv_conf.h"
+#include "lvgl_riot.h"
+#include "disp_dev.h"
+
+#ifndef LVGL_TASK_THREAD_PRIO
+#define LVGL_TASK_THREAD_PRIO   (THREAD_PRIORITY_MAIN + 1)
```

Comment:
so, lower priority (higher value) than main?

Reply:
I tried both versions and it works the same. Not sure which one is best.

Reply:
ok. I guess it depends on what's actually running in main.